src/current/_includes/releases/v23.2/v23.2.5.md

```diff
@@ -0,0 +1,68 @@
+
+<h3 id="v23-2-5-bug-fixes">Bug fixes</h3>
+
+- Fixed a slow memory leak when opening many new connections. This bug was introduced in 22.2.9 and 23.1.0. [#121055][#121055]
+- Fixed a bug that occurred when using [`ALTER TABLE`]({% link {{ page.version.version }}/alter-table.md}) to drop and re-add a CHECK constraint with the same name. [#121055][#121055]
+- [Sequence]({% link {{ page.version.version }}/create-sequence.md %}) options `MINVALUE` and `MAXVALUE` automatically adjust to new types bounds. This mirrors the behavior of postgreSQL. [#121309][#121309]
+- Fixed a bug that could prevent timeseries graphs shown on the DB Console SQL Activity [Statement Details]({% link {{ page.version.version }}/statement-details.md %}) page from rendering correctly when specifying a custom time range. [#121383][#121383]
+- Fixed a bug present since at least v21.1 that could lead to incorrect evaluation of an `IN` expression with:
+    - `INT2` or `INT4` type on the left side, and
+    - Values on the right side that are outside of the range of the left side.
+
+    [#121953][#121953]
+- Fixed a leak in reported memory usage (not the actual memory usage) by the internal memory accounting system, the limit for which is configured via the `--max-sql-memory flag` when a long-running sessions issues hundreds of thousands or more transactions. This reporting bug could cause `root: memory budget exceeded` errors for other queries. The bug was introduced in v23.1.17 and f23.2.3. [#121950][#121950]
+- Fixed a bug introduced in v23.2.4 that could prevent collection of table statistics on tables that have virtual computed columns of user-defined type when the newly-introduced [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) `sql.stats.virtual_computed_columns.enabled` is set to `true` (defaults to `false`). The setting is introduced in 23.2.4  and is disabled by default, so only this particular version is affected with non-default setting. [#122319][#122319]
+- Fixed a bug where a [`GRANT ...ON ALL TABLES`]({% link {{ page.version.version }}/grant.md %}) statement could fail if a sequence existed that did not support the privilege being granted. [#122034][#122034]
+- Fixed an existing bug where an unused value cannot be dropped from an [enum]({% link {{ page.version.version }}/enum.md %}) if the enum itself is referenced by a [user-defined function]({% link {{ page.version.version }}/user-defined-functions.md %}). A value can now be dropped from an enum as long as the value itself is not being referenced by any other data element, including a user-defined function. [#121237][#121237]
+
+<div class="release-note-contributors" markdown="1">
+
+<h3 id="v23-2-5-contributors">Contributors</h3>
+
+This release includes 79 merged PRs by 33 authors.
+We would like to thank the following contributors from the CockroachDB community:
```

Comment:
this section should end after "33 authors."

we no longer list the "thank the following contributors" stuff, since internal folks use different non-work emails and show up in this list - probably other reasons too but that's the one I remember